lib/lte_link_control/lte_lc_helpers.h

```diff
@@ -307,6 +315,19 @@ int parse_coneval(const char *at_response, struct lte_lc_conn_eval_params *param
  */
 int parse_mdmev(const char *at_response, enum lte_lc_modem_evt *modem_evt);
 
+/* @brief Parse a RAI response and extract RAI configuration.
+ *
+ * @param at_response AT response.
+ * @param rai_cfg RAI configuration.
+ *
+ * @return Zero on success, negative errno code on failure.
+ *
+ * @retval 0 Parsing succeeded.
+ * @retval -EINVAL If invalid parameters are provided.
+ * @retval -EIO If the AT response is not a valid RAI response.
```

Comment:
Have you observed this error in practice? This error (from `at_parser`) indicates parsing beyond the limits of the AT response string.

Reply:
`at_parser` can return quite many different errors, not sure which ones are possible in this case. Do we need to try to document those here, after all, this is an internal API? I think at least `-EIO` could be removed.

Reply:
In unit tests I am able to get `-EIO` by giving RAI notification without some of the parameters so we try to read parameters beyond the limits. Obviously, this would be a modem bug if it would send such a notification.

Reply:
And generally, I this whole file has mixed styles. Some functions lists possible error codes while others don't. And I doubt there are some where the list is not complete.
I'm not planning to clean this up in this PR.

Reply:
I've removed the comments related to return values as we are not sure if they are complete and they are not documented for all other functions either. This is an internal API anyway.